tests/test_pipeline_generation.py

```diff
@@ -8,13 +8,8 @@ def test_generate_pipeline(pipeline):
     session = pipeline["session"]
     Equipment = pipeline["Equipment"]
 
-    subject_tbl, *_ = session.Session.parents(as_objects=True)
-
-    # test elements connection from lab, subject to Session
-    assert subject_tbl.full_table_name == subject.Subject.full_table_name
-
-    # test elements connection from Session to scan, imaging
-    session_tbl, equipment_tbl, _ = scan.Scan.parents(as_objects=True)
+    assert schema.Table.full_table_name in schema.OtherTable.parents()
```

Comment:
Thanks @virenvarma007.  The code snippet provided in #74 was a template and will need to be adapted to the specific tables being tested.